Libraries/LibWeb/CSS/StyleValues/CSSColorValue.h

```diff
@@ -15,10 +15,15 @@
 
 namespace Web::CSS {
 
+enum class LegacySyntax {
+    Yes,
+    No,
+};
```

Comment:
This might have been nicer as `Syntax::Legacy/Syntax::Modern` (or even `ColorSyntax::...`) but that's debatable. :bikeshed: I don't have a strong enough opinion to make you change it.

Reply:
Good idea, I like the `ColorSyntax::{Legacy/Modern}` naming